src/utils/table.ts

```diff
@@ -1,5 +1,7 @@
 import { Table } from "types/Table";
 
+const DEFAULT_COLUMNS_COUNT = 3;
```

Comment:
IMHO it's worth keeping this const in `src/settings.ts`. WDYT?

Reply:
Makes sense to do so to me.

Reply:
it's already there -- appeared when I merged changes from staging. `DEFAULT_TABLE_COLUMNS`. 🙃 

(but haven't tested the script after that merge)